pager/src/main/java/com/google/accompanist/pager/PagerState.kt

```diff
@@ -106,7 +118,16 @@ class PagerState(
             )
         }
 
-    internal var currentLayoutPageSize by mutableStateOf(0)
+    internal inline val currentLayoutPageSize: Int
```

Comment:
Size makes me think 2d, but this is the width/height depending upon orientation right? Not sure I have a better suggestion tho.